### ISSUE TITLE
Exploiting Cloze Questions for Few Shot Text Classification and Natural Language Inference

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#115]
# TL;DR

They propose a PET  that use a mask language model and  multiple patterns of sentence with  <MASK> token to answer, and treats the ensemble of <MASK> output as a label. It works well even with a small data set.
<img width="381" alt="AG's News" src="https://user-images.githubusercontent.com/24524018/93409018-e362a400-f8d0-11ea-93af-0f0c94d750e7.png">

# Why it matters:

# Paper URL
https://arxiv.org/abs/2001.07676

# Submission Dates(yyyy/mm/dd)
2020/01/21

# Authors and institutions
Timo Schick, Hinrich Schütze
- Center for Information and Language Processing, LMU Munich, Germany
- Sulzer GmbH, Munich, Germany

# Methods

# Results

# Comments
